### ISSUE TITLE
'Unable to enable shell extention for .rar & .7z

### ISSUE THREAD

**Issue by Pandorasheaart** [Nexus-Mods/Nexus-Mod-Manager#835]
I am attempting to mod Skyrim. Oldrim, not SSE
Nexus was having the 'exception in the script error' 
I attempted to add shell extentions. 
Now I get the message that 'Shell extention assotiation for .rar & .7z has been removed by some other process - do you want to restore it?' 
When I click 'yes' it gives me another pop up message 'Unable to restore shell extention' 
I am running as admin. I also use NMM for FO4 

Load order image is below as well as images of the pop-ups

TIA - Pandora X.x
- NMM version: _0.70.7_
![ug](https://user-images.githubusercontent.com/42595417/59437139-397f9800-8dae-11e9-84c7-ff20a93488b5.PNG)
![v](https://user-images.githubusercontent.com/42595417/59437142-3b495b80-8dae-11e9-8ea5-8f3a72169b5b.PNG)
![Capture](https://user-images.githubusercontent.com/42595417/59436669-48197f80-8dad-11e9-8991-c652ae5c5d61.PNG)

**Comment by squid-box** [comment]
Could you start NMM with forced trace logging, re-create the error, and then upload the created trace log?

Check this wiki page on trace logging: https://github.com/Nexus-Mods/Nexus-Mod-Manager/wiki/Bug-reporting#trace-log

**Comment by Pandorasheaart** [comment]
[TraceLog20190613154507.txt](https://github.com/Nexus-Mods/Nexus-Mod-Manager/files/3288200/TraceLog20190613154507.txt)
here you go, this is only for the 'exception occured in the script' issue though. NMM does not create a trace log for the unknown error and 'association removed by another process' issues

**Comment by squid-box** [comment]
There were still some log entries for the shell extensions, so I've added a fix for it in PR #841. Once that is merged it should be included in the coming release.

If you need a work-around until then, have some other program associate with .7z and .rar files, and NMM can handle it afterwards.

**Comment by Pandorasheaart** [comment]
So, manual download and then have NMM Install?

**Comment by squid-box** [comment]
Not sure I follow - having read your issue again it seems the shell extension is not the only problem you're having? What's the "exception occurred in script" you're mentioning? There's nothing about it in the trace log you've attached - only the failing to create file extension associations.

My solution will fix the problem with adding extensions - which is caused by there not being any existing associations for .7z or .rar on your computer. If you were to eg. install 7-Zip and have it associate itself with .7z and .rar files, then NMM could add them in version 0.70.7, otherwise you'll have to wait for 0.70.8 to be released.

**Comment by madopapanikolaou** [comment]
Hey I’m having the same issue and I’m using the 0.70.9 version , did you ever find a way to fix it

**Comment by Pandorasheaart** [comment]
No. I left it. Went back and realized I didn't have the script extender
installed.
